docs/ide/editorconfig-code-style-settings-reference.md

```diff
@@ -104,16 +113,16 @@ error | When this style is not being followed, show a compiler error.
 dotnet_style_qualification_for_field = false:suggestion
 ```
 
-### <a name="this_and_me_properties">Properties</a>
+### <a name="this_and_me_properties">Properties (IDE0003/IDE0009) </a>
 
 |  Option Name | `dotnet_style_qualification_for_property` |
 | ------------- |:-------------:|
 | **Applicable Languages** | C# and Visual Basic

```

Comment:
:bulb: It would be good to list the default value and the first release that supported the option here as well.